src/parser/jobs/war/modules/SurgingTempest.tsx

```diff
@@ -41,3 +65,62 @@ export class SurgingTempest extends Analyser {
 		return (statusUptime / fightUptime) * 100
 	}
 }
+
+export class StormsEye extends CoreGauge {
+	static override handle = 'stormseye'
+
+	@dependency private suggestions!: Suggestions
+
+	private surgingTempest = this.add(new TimerGauge({
+		maximum: 60,
+	}))
+
+	private earlyRefreshCount = 0
+
+	override initialise() {
+		super.initialise()
+		this.addEventHook(
+			filter<Event>()
+				.source(this.parser.actor.id)
+				.action(this.data.matchActionId(SURGING_TEMPEST_GENERATORS)),
+			(_) => this.extendSurgingTempest(SURGING_TEMPEST_GENERATION_AMOUNT)
+		)
+
+		this.addEventHook(
+			filter<Event>()
+				.source(this.parser.actor.id)
+				.action(this.data.matchActionId(SURGING_TEMPEST_EXTENDERS)),
+			(_) => this.extendSurgingTempest(SURGING_TEMPEST_EXTENSION_AMOUNT, true)
+		)
+		this.addEventHook('complete', this.onComplete)
+	}
+
+	private extendSurgingTempest(amount: number, onlyIfRunning ?: boolean) {
+		onlyIfRunning === undefined ? false : onlyIfRunning
+
+		if (this.surgingTempest.remaining > SURGING_TEMPEST_EARLY_REFRESH_GRACE) {
+			this.earlyRefreshCount++
+		}
+
+		if ((this.surgingTempest.expired || this.surgingTempest.paused) && !onlyIfRunning) {
+			this.surgingTempest.start()
+		}
+		this.surgingTempest.extend(amount, onlyIfRunning)
+	}
+
+	private onComplete() {
+		if (this.earlyRefreshCount > 0) {
```

Comment:
This also isn't necessary, tiered suggestions only show when their minimum tier is hit, so since your lowest tier isn't 0 (which would always show it so kinda derp), this conditional has no effect other than making the code more indented.